AutomatorServer/android/app/src/androidTest/java/pl/leancode/automatorserver/PatrolAutomator.kt

```diff
@@ -47,15 +48,20 @@ private fun fromUiObject2(obj: UiObject2): Contracts.NativeView {
 }
 
 class PatrolAutomator private constructor() {
+    // Timeout in milliseconds
```

Comment:
You could create a value type for milliseconds – this will make users pass milliseconds without having to read this comment

Reply:
I get `Long` from protobuf and the APIs I use only accepted `Long`, and I am the only user (these are not user-facing APIs). I don't see a benefit to this.

Reply:
I prefer a better var name instead of comment

Reply:
Renamed to `timeoutMillis`, thanks!